docs/tutorials/cli-beginners-guide.rst

```diff
@@ -0,0 +1,155 @@
+=========================================
```

Comment:
**issue**: this file doesn't seem to be included anywhere, so it won't be visible to the user. There's a related build warning:
```
/Users/alex/Code/zapata/orquestra-docs/docs/core/sdk/tutorials/cli-beginners-guide.rst: WARNING: document isn't included in any toctree
```

Did you plan to add it to the `docs/tutorials/index.rst`?